test/export/test_export.py

```diff
@@ -2446,6 +2490,36 @@ def g(x):
         ):
             constraints = [dynamic_dim(inp_for_g, 0)]
 
+    def test_mark_and_auto_dynamic(self):
```

Comment:
added test with mark_dynamic. Semantics haven't changed, this is just a test case where AUTO & mark_dynamic result in the same program